test/e2e/mmi/helpers/oauth-api-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { type BrowserContext } from '@playwright/test';
+
+export class OAuthAPIClient {
+  readonly context: BrowserContext;
+
+  readonly baseUrl: string;
+
+  constructor(context: BrowserContext) {
+    this.context = context;
+    this.baseUrl = process.env.MMI_E2E_AUTH0_TOKEN_URL as string;
+  }
+
+  async getToken() {
+    if (process.env.AUTH0_ACCESS_TOKEN) {
```

Comment:
From where does the env variable value comes from?

Reply:
Locally a .env file is required. In the pipeline all env vars come from CircleCI environment variables setup.

Reply:
Ah got it! 👍🏼